src/components/TDViewer/TDViewer.jsx

```diff
@@ -98,15 +127,141 @@ export default function TDViewer() {
         metaData = tdJSON;
     }
 
-    const alreadyRenderedKeys = ["id", "properties", "actions", "events", "description", "title",];
+    const alreadyRenderedKeys = ["id", "properties", "actions", "events", "forms", "description", "title",];
 
-    const attributeListObject = buildAttributeListObject(tdJSON.id ? {id: tdJSON.id} : {}, tdJSON, alreadyRenderedKeys);
+    const attributeListObject = buildAttributeListObject(tdJSON.id ? { id: tdJSON.id } : {}, tdJSON, alreadyRenderedKeys);
 
     const attributes = Object.keys(attributeListObject).map(x => {
         return <li key={x}>{x} : {JSON.stringify(attributeListObject[x])}</li>
     });
 
 
+    const sortKeysInObject = (kind) => {
+        const ordered = {};
+        const toSort = Object.keys(tdJSON[kind]).map(x => {
+            return { key: x, title: tdJSON[kind][x].title }
+        })
+        if (sortorder === 'asc') {
+            toSort.sort(function (a, b) {
+                var nameA = a.title ? a.title.toUpperCase() : a.key.toUpperCase();
+                var nameB = b.title ? b.title.toUpperCase() : b.key.toUpperCase();
+                if (nameA < nameB) {
```

Comment:
Have you thought of using `String.localeCompare()`?

I'd also sort it once and reverse only if necessary.

Reply:
Nice hint. 
Changed to use it 😄